app/src/main/java/io/homeassistant/companion/android/sensors/SettingsSensorManager.kt

```diff
@@ -0,0 +1,73 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.provider.Settings
+import android.util.Log
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.common.R as commonR
+
+class SettingsSensorManager : SensorManager {
+    companion object {
+        private const val TAG = "ScreenBrightness"
+
+        val screenBrightness = SensorManager.BasicSensor(
+            "screen_brightness",
+            "sensor",
+            commonR.string.basic_sensor_name_screen_brightness,
+            commonR.string.sensor_description_screen_brightness,
+        )

```

Comment:
The sensor should have a stateless icon so something is shown before enabling it